src/__tests__/components/fields/location-field/location-field.spec.tsx

```diff
@@ -1144,6 +1144,16 @@ describe("location-input-group", () => {
 						it.todo("should show both static map and input value and dismiss location modal");
 					});
 				});
+
+				describe("when a banner text is provided", () => {
```

Comment:
can there also be a test for not showing banner text? do the components have testids appended to them?

Reply:
Sure will add one. The location-picker component renders a `LocationPickerWrapper` which has a testid attached to it but not for the components within the wrapper.

Reply:
can probably have a testid for the banner component and test for it

Reply:
Added a testid to the banner and added a unit test

Reply:
Combined both test cases into one test suite. Thanks!